app/models/merchant.rb

```diff
@@ -7,8 +7,6 @@ class Merchant < ApplicationRecord
   has_many :customers, through: :invoices
   has_many :invoice_items, through: :invoices
 
-  include Intelligence
-
   def single_merchant_revenue
     invoices
       .select("sum(invoice_items.unit_price * invoice_items.quantity/100) AS revenue")
```

Comment:
@adam-conway Have you reseeded as of the changes that I made to import.rake? I see here that you divide by 100.

Reply:
@agpiermarini It looks like the unit price in my db for invoice items is as a big decimal so i'll have to re-seed

Reply:
@agpiermarini Just reseeded and adjusted everything